src/App.tsx

```diff
@@ -15,37 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
```

Comment:
move ur `enum` to a separate file - `utils/enums/sortType.ts`

Reply:
I think it would be better to create a type catalog and keep all types (types, interfaces, enums, etc.) to be sorted in a SortTypes file, because then it would be easy to search for the types you need in one file named after the feature it is used in.

Reply:
`enum` is a type that can be used in dozens of places in a project. NOT only in 1 function. Therefore, they are placed in a separate directory
`enum` - not bound to 1 function